File: VB/E4434/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Namespace E4434
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
			AddRows(5)
		End Sub

		Private Sub AddRows(ByVal rowCount As Integer)
			For i As Integer = 0 To rowCount - 1
				unboundGridView1.AppendRow(New Object() { String.Format("String {0}", i), i, DateTime.Today.AddDays(i) })
			Next i
		End Sub
	End Class
End Namespace

File: CS/E4434/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace E4434 {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
            AddRows(5);
        }

        private void AddRows(int rowCount) {
            for (int i = 0; i < rowCount; i++) {
                unboundGridView1.AppendRow(new object[] { string.Format("String {0}", i), i, DateTime.Today.AddDays(i) });
            }
        }
    }
}
